gym/spaces/dict.py

```diff
@@ -44,7 +46,7 @@ class Dict(Space[TypingDict[str, Space]], Mapping):
     def __init__(
         self,
         spaces: dict[str, Space] | None = None,
-        seed: dict | int | None = None,
+        seed: Optional[int | seeding.RandomNumberGenerator] = None,
```

Comment:
Why remove the option of passing a dict of seeds here?

Reply:
+1 on the question (also for the sequence of ints later on in the code)